src/nibpkg/read.nim

```diff
@@ -8,11 +8,13 @@ type Read* = object
     ## key of svid, count of supporting kmers
     compatible_SVs* : CountTable[uint32]
 
-proc process_read*(s : string, idx : svIdx, k:int=25): Read =
+proc process_read*(s : string, idx : svIdx, k:int=25, spacedSeeds : bool = false, space : int = 50): Read =
```

Comment:
Note, our default kmer size is 25 here and 21 over in ref_kmers. That's going to cause issues.

Can we set a macro in the package somehow that makes the default size consistent across nib?